src/ResourceManager/Sql/Commands.Sql.Test/Commands.Sql.Test.csproj

```diff
@@ -79,7 +79,8 @@
     </Reference>
     <Reference Include="Microsoft.Azure.Management.Sql, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Sql.1.13.0-preview\lib\net452\Microsoft.Azure.Management.Sql.dll</HintPath>
+      <!--<HintPath>..\..\..\packages\Microsoft.Azure.Management.Sql.1.10.0-preview\lib\net452\Microsoft.Azure.Management.Sql.dll</HintPath>-->
```

Comment:
If I'm not mistaken, this SDK has now been published, so you can update to 1.13.1 instead of your local package (this should fix most build issues).

Reply:
Update the SDK package to 1.15.0-preview which including our changes.